flytekit/core/context_manager.py

```diff
@@ -373,11 +374,22 @@ def get(
         if not get_plugin().secret_requires_group():
             group, group_version = None, None
 
-        env_var = self.get_secrets_env_var(group, key, group_version)
+        env_prefixes = [self._env_prefix]
+
+        # During local execution check for the key without a prefix
+        is_local_execution = os.getenv("FLYTE_INTERNAL_EXECUTION_ID") is None
```

Comment:
could we convert this to utility function as i think it could be useful in other places to know if you are in local mode or not . But don't we have --remote flag that we can use to determine the same